lib/rom/sql/attribute.rb

```diff
@@ -154,6 +154,20 @@ def is(other)
         __cmp__(:'=', other)
       end
 
+      # Return a boolean expression with `!=` operator
+      #
+      # @example
+      #   users.where { id.not(1) }
+      #
+      #   users.where(users[:id].not(1))
+      #
+      # @param [Object] other Any SQL-compatible object type
+      #
+      # @api public
+      def not(other)
+        __cmp__(:'!=', other)
+      end
```

Comment:
You can make it more flexible by doing `~is(other)` which uses sequel's negation operator (not sure why it's not called `!` though)